### ISSUE TITLE
Just Cause 2 Multiplayer Support

### ISSUE THREAD

**Issue by Ruok2bu** [Austinb/GameQ#129]
Hi Austin,

I got info directly from the developers of JC2MP about their server query protocol.  Here's what the dev told me:

We originally only supported the OGP query protocol, however with the recent addition of Steam support, we added partial support to Steam queries too.

Given that Steam's query protocol supports a maximum player count of 255, it is better to use OGP (http://www.open-game-protocol.org/) to query our servers. If you do choose Steam though, we have placed our own player count in the "map" field of the protocol. It's a little hack, but ensures you can obtain player counts using both query systems.

**Comment by Ruok2bu** [comment]
The main problem with the steam protocol is that the player count reported looks like this (for a 100 slot server):

Players: 0/100

So if you use the Steam protocol, can you strip out "Players: "  and the /MAXPLAYERS so that the fields match other games in GameQ?

**Comment by Austinb** [comment]
I have got this working via source. Didnt want to spend the time building for OGP when there are only 3 games on their list (http://www.open-game-protocol.org/gamelist.php)..

I did notice a bug in the rules response.  Servers I tested have the maxplayers and players in the rules response.  There is an issue where the players rule is actually ^Alayers. I am pretty sure this is a bug in the rules response from the server as it should be Players.  So this will show up as a key in the response as "^Alayers" instead of "Players".  They should try to fix that. 

Also if the rules have the player information there is no real reason to use the map field right?  Also is it possible to pass the map via the rules?
